wapitiCore/definitions/file.py

```diff
@@ -24,6 +24,8 @@
 NAME = _("Path Traversal")
 SHORT_NAME = NAME
 
+WSTG_CODE = ["WSTG-ATHZ-01"]
+
```

Comment:
Please update OWASP reference to https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include